documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -69,30 +108,81 @@ value.converter.apicurio.registry.url=http://apicurio:8080/api
 value.converter.apicurio.registry.global-id=io.apicurio.registry.utils.serde.strategy.AutoRegisterIdStrategy
 ----
 
-Note that Kafka Connect internally always uses the JSON key/value converters for storing configuration and offsets.
+Internally, Kafka Connect always uses JSON key/value converters for storing configuration and offsets.
+
+// Type: procedure
+// Title: Deploying connectors that use Avro in {prodname} containers
+// ModuleID: deploying-connectors-that-use-avro-in-debezium-containers
+== Deploying with {prodname} containers
+
+In your environment, you might want to use a {prodname} pre-built container to deploy a {prodname} connector that uses Avro serializaion. Follow the procedure here to do that. In this procedure, you build a {prodname} image in which {prodname} uses the Avro converter.  
+
+.Prerequisites
 
-== {prodname} Container Images
+* You have the required permissions on a Kafka cluster.
+* You downloaded the {prodname} connector plug-in that you want to deploy with Avro serialization.
 
-Deploy an Apicurio Registry instance (this example uses a non-production in-memory instance):
+.Procedure
 
-[source,subs="+attributes"]
+. Deploy an instance of {registry}. 
++
+ifdef::community[]
+The following example uses a non-production, in-memory, {registry}  instance:
++
+[source,subs="attributes"]
 ----
 docker run -it --rm --name apicurio \
     -p 8080:8080 apicurio/apicurio-registry-mem:{apicurio-version}
 ----
+endif::community[]
+ifdef::product[]
+.. Choose the OpenShift project in which you want to deploy the {prodname} connector. In the following command, `$NAMESPACE` represents your project.  
+.. Deploy the latest published {registry} operator by running the following command:
++
+`curl -sSL https://raw.githubusercontent.com/apicurio/apicurio-registry-operator/master/docs/resources/install.yaml | sed "s/{NAMESPACE}/$NAMESPACE/g" | oc apply -f -`
```

Comment:
Is this the right resource for downstream?

Reply:
I asked Steve McCarthy, to point me to the downstream resource and this is what he pointed me to. I don't know his GitHub user name. I'll find out and tag him here to confirm.

Reply:
Ok, let's clarify that. In particular using "master" seems really odd for product, I'd expect a fixed reference instead of a live branch.

Reply:
Adding @smccarthy-ie , to help us straighten this out.

Reply:
@TovaCohen @gunnarmorling, OK, some confusion around upstream/downstream product names. You were asking about Apicurio Registry (upstream) installation, but Service Registry is the downstream product :)  

The latest downstream Service Registry docs show using the OperatorHub in the OpenShift UI. I don't know the command line equivalent. 

@jsenko Can you tell us what is the downstream equivalent of `curl -sSL https://raw.githubusercontent.com/apicurio/apicurio-registry-operator/master/docs/resources/install.yaml | sed "s/{NAMESPACE}/$NAMESPACE/g" | oc apply -f -`

Reply:
I replaced these instructions with a cross-reference to follow the instructions in Getting Started with Service Registry, "Installing Service Registry from the OperatorHub".
@gunnarmorling  Let me know if you think we also need to provide a command line alternative.